src/SFA.DAS.EmployerAccounts.Web/Views/EmployerTeam/V2/ApprenticeshipDetails.cshtml

```diff
@@ -0,0 +1,27 @@
+﻿@model SFA.DAS.EmployerAccounts.Web.ViewModels.AccountDashboardViewModel
+
+    <div class="das-panel">
+        <h3 class="govuk-heading-l">Apprentice details </h3>
+
+        <dl class="das-definition-list das-definition-list--table">
+            <dt class="das-definition-list__title">Name</dt>
+            <dd class="das-definition-list__definition">@Model.AccountViewModel.Organisations.FirstOrDefault().Cohorts.FirstOrDefault().Apprenticeships.FirstOrDefault().FirstName+" "+@Model.AccountViewModel.Organisations.FirstOrDefault().Cohorts.FirstOrDefault().Apprenticeships.FirstOrDefault().LastName</dd>
+
+            <dt class="das-definition-list__title das-definition-list__title--spacer">Cohort reference</dt>
+            <dd class="das-definition-list__definition das-definition-list__definition--spacer">@Model.AccountViewModel.Organisations.FirstOrDefault().Cohorts.FirstOrDefault().Id</dd>
+
+            <dt class="das-definition-list__title">Training course</dt>
+            <dd class="das-definition-list__definition">@Model.AccountViewModel.Organisations.FirstOrDefault().Cohorts.FirstOrDefault().Apprenticeships.FirstOrDefault().CourseName</dd>
+
+            <dt class="das-definition-list__title">Start date</dt>
+            <dd class="das-definition-list__definition">@Model.AccountViewModel.Organisations.FirstOrDefault().Cohorts.FirstOrDefault().Apprenticeships.FirstOrDefault().StartDate.ToString()</dd>
```

Comment:
The start and end dates should be in e.g. "January 2020" format.

You can use `ToString("MMMM yyyy")`.

Reply:
👍

Reply:
Done